include/ergw_aaa.hrl

```diff
@@ -0,0 +1,23 @@
+%% Copyright 2021, Travelping GmbH <info@travelping.com>
+%%
+%% This program is free software: you can redistribute it and/or modify
+%% it under the terms of the GNU Lesser General Public License as
+%% published by the Free Software Foundation, either version 3 of the
+%% License, or (at your option) any later version.
+%%
+%% This program is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied warranty of
+%% MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+%% GNU Lesser General Public License for more details.
+%%
+%% You should have received a copy of the GNU Lesser General Public License
+%% along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+-record(peer, {
```

Comment:
we don't want to export the record to the outside of ergw_aaa. Please move this into src/ergw_aaa_diameter_srv.hrl

Reply:
Moved to exist file `src/ergw_aaa_internal.hrl`